inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -112,15 +118,22 @@ def calculate_rmsf_trajectory(self,
         plt.title("RMSF per residue")
         plt.grid(True)
         plt.show()
+        duration = time.time() - start_time
+        print(f"RMSF calculation took {duration:.2f} seconds.")
         return rmsf_values
```

Comment:
It seems plotting would take the output of this method and some other parameters (time) and plot this in whatever way we want. Plotting inside this method seems a bit weird.